model/metabolites.tsv

```diff
@@ -183,7 +183,7 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "MAM00095p"	"MAM00095"	"c226coa"	"C16169"	""	"CHEBI:65132"	""	"LMFA07050050"	"CE4827"	""	"CE4827;c226coa"	"MNXM3234;MNXM91778"	"m00095p"	"m00095p"
 "MAM00095r"	"MAM00095"	"c226coa"	"C16169"	""	"CHEBI:65132"	""	"LMFA07050050"	"CE4827"	""	"CE4827;c226coa"	"MNXM3234;MNXM91778"	"m00095r"	"m00095r"
 "MAM00096c"	"MAM00096"	""	"C02100"	""	""	""	""	""	""	"M00096"	"MNXM92760"	"m00096c"	"m00096c"
-"MAM00097s"	"MAM00097"	"gluala"	"C03740"	"HMDB06248"	"CHEBI:50619"	"440103"	""	""	""	"gluala"	"MNXM59390"	"m00097s"	"m00097s"
+"MAM00097s"	"MAM00097"	"gluala"	""	"HMDB06248"	"CHEBI:50619"	"440103"	""	""	""	"gluala"	"MNXM59390"	"m00097s"	"m00097s"
```

Comment:
To address the current conflict with `develop`, could you please change `MAM00097s` (in the first column) to `MAM00097e`

Reply:
done, please see https://github.com/SysBioChalmers/Human-GEM/pull/358/commits/041ad028d203e6015589d8e901f2bf0ea07bb0be